{{cookiecutter.repo_name}}/ci/templates/.travis.yml

```diff
@@ -1,9 +1,12 @@
-language: python
-python: '3.5'
 sudo: false
+language: python
+python: "3.4"
 env:
+{% if cookiecutter.c_extension_support|lower == "yes" %}
   global:
-    LD_PRELOAD=/lib/x86_64-linux-gnu/libSegFault.so
+    - LD_PRELOAD=/lib/x86_64-linux-gnu/libSegFault.so
+    - SEGFAULT_SIGNALS="abrt segv"
```

Comment:
Interesting, didn't knew about this env var. But shouldn't it be `SEGFAULT_SIGNALS=all`?